.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,15 @@
-## What
+What
```

Comment:
Not a fan of the nice # syntax?

Reply:
No, see commit: https://github.com/alphagov/paas-admin/pull/68/commits/9347483d3f010c13c2bbaae030867f4425e1d571